### ISSUE TITLE
Test Data Preparation

### ISSUE THREAD

**Issue by sundeep-co-in** [transtats/transtats#106]
##### ISSUE TYPE

 - Feature Idea

##### OS / ENVIRONMENT

```
devel
```

##### SUMMARY

This is really good idea (thanks @juhp) to keep sets of test-data to automate feature specific environment. May be configuration based. This will certainly save lot of time.

**Comment by sundeep-co-in** [comment]
@spathare21 you have something to add here?

**Comment by spathare21** [comment]
@sundeep-co-in nope, as discussed we will create test data for deploying test env

**Comment by sundeep-co-in** [comment]
temporarily https://github.com/transtats/transtats/pull/94/commits/da64316af3162f1f97959b89b674bb703ba80c28 should handle this!

**Comment by sundeep-co-in** [comment]
@spathare21 how far we have reached here?
